### DISCUSSION TITLE
Is there any way to insert page between two pages in the fixed-sized document layout?

### DISCUSSION THREAD

**Discussion by mahdi4261** [flxzt/rnote#1316] [Q&A] [answered]
I only found the "Insert Vertical Space" tool. This tool cases trouble if there are multiple pages of note after like this:

[Screencast From 2024-12-06 22-58-02.webm](https://github.com/user-attachments/assets/651d30d2-37f9-4653-ae63-239925608a03)

**Comment by Doublonmousse** [comment, marked as answer]
Yes, for now this is the only option to add a page in between two pages. For now, there isn't really a concept of page in rnote, everything is treated as a continuous canvas (with or without limits depending on the modes).

There was some added functionality to the vertical space tool (not yet released) to limit the tool's effect vertically or horizontally (so that you can add space inside of a page without modifying others) in #1116
